pkg/operators/ebpf/ebpf.go

```diff
@@ -325,6 +353,78 @@ func (i *ebpfInstance) init(gadgetCtx operators.GadgetContext) error {
 	return nil
 }
 
+func (i *ebpfInstance) addExtraInfo(gadgetCtx operators.GadgetContext) error {
```

Comment:
I would move this function and the 3 types to a new file `pkg/operators/ebpf/extrainfo.go` since `ebpf.go` is already big. That will also be better for future PRs adding more extra-infos.